src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1649,6 +1649,13 @@ namespace ScheduleManager {
                 FileIntervalInterpolated = true;
             }
 
+            if (lAlphaBlanks(6)) Alphas(6) = "YES";
+            if ((Alphas(6)) == "YES") {
+                state.dataScheduleMgr->Schedule(SchNum).UseDaylightSaving = true;
+            } else if ((Alphas(6)) == "NO") {
+                state.dataScheduleMgr->Schedule(SchNum).UseDaylightSaving = false;
+            }
+
```

Comment:
If Alphas(6) field is a "Yes" or a blank, assign a "true" value to the UseDaylightSaving boolean, otherwise assign a "false" value. This boolean is then used in an if statement to decide whether or not to shift the schedule by one hour forward.